report/renderer/category-renderer.js

```diff
@@ -144,6 +144,35 @@ export class CategoryRenderer {
     return auditEl;
   }
 
+  /**
+   * Inject the final screenshot next to the score gauce of the first category (likely Performance)
+   * @param {HTMLElement} categoriesEl
+   * @param {LH.ReportResult['audits']} audits
+   * @param {Element?} scoreScaleEl
+   */
+  injectFinalScreenshot(categoriesEl, audits, scoreScaleEl) {
+    // TODO: Use full-page-screenshot instead as that's always gathered, regardless of category filter
+    const audit = audits['final-screenshot'];
+    if (!audit || audit.scoreDisplayMode === 'error') return null;
+    const details = audit.details;
+    if (!details || details.type !== 'screenshot') return null;
+    const finalScreenshotDataUri = details.data;
+    const finalSSimg = this.dom.createElement('img', 'lh-final-ss-image');
```

Comment:
maybe rename to `imgEl` ?

Reply:
bump: this is an element, so it'd be nice to name it like all element variables (ending in `El`).